internal/cli/iam/organizations/apikeys/accesslists/access_lists.go

```diff
@@ -36,3 +42,68 @@ func Builder() *cobra.Command {
 
 	return cmd
 }
+
+// shouldUseWhitelist returns true when it is service Ops Manager and all versions below 5.0
+// returns false when Atlas, Cloud Manager or Ops Manager 5+.
+func shouldUseWhitelist(s store.ServiceVersionGetter) (bool, error) {
+	if config.Service() != config.OpsManagerService {
+		return false, nil
+	}
+
+	version, err := s.GetServiceVersion()
```

Comment:
how do you feel about having all this from `s.GetServiceVersion()` to `sv, err := semver.NewVersion(newVersion)` as a separate method?